qemu/tests/blockdev_commit_install.py

```diff
@@ -0,0 +1,57 @@
+import time
+import logging
+import random
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest.tests import unattended_install
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitInstall(BlockDevCommitTest):
+    def configure_system_disk(self, tag):
+        pass
+
+
+def run(test, params, env):
+    """
+    Block commit base Test
+
+    1. Install guest
+    2. create 4 snapshots during guest installation
+    3. commit snapshot 3 to base
+    4. installation can be finished after commit
+    """
+    def tag_for_install(vm, tag):
+        if vm.serial_console:
+            serial_output = vm.serial_console.get_output()
+            if tag in serial_output:
+                return True
+        logging.info("vm has not started yet")
+        return False
+
+    block_test = BlockdevCommitInstall(test, params, env)
+    args = (test, params, env)
+    bg = utils_test.BackgroundTest(unattended_install.run, args)
+    bg.start()
+    if bg.is_alive():
+        tag = params.get("tag_for_install_start", "Starting Login Service")
```

Comment:
add tag_for_install_start = "Starting Login Service" to cfg file?

Reply:
no need, we have set the default value. We need to add it to cfg when the value is not the default.

Reply:
actually I suggest that could like this:
cfg:
`tag_for_install_start = "Starting Login Service"` 
py:
tag = params["tag_for_install_start"]

since for this case must need a tag for installing.
if the tag_for_install_start is changed in the rhel9 or other version.
might do :
```
RHEL9:
    tag_for_install_start = "XXXXXXXX"
```
It's not a big issue, up to you.

Reply:
Will not change it, if needed, will set it in cfg. thanks for all your suggestion.